api/v1alpha1/virtualmachine_types.go

```diff
@@ -185,6 +189,26 @@ type TCPSocketAction struct {
 	Host string `json:"host,omitempty"`
 }
 
+// The guest heartbeat status.
+type GuestHeartbeatStatus string
+
+// See govmomi.vim25.types.ManagedEntityStatus
+const (
+	// Intermittent heartbeat. May be due to guest load.
+	YellowStatus GuestHeartbeatStatus = "yellow"
```

Comment:
Should we change the name to reflect that its related to Heartbeat? Seeing `YellowStatus` might not make it clear that we are talking about Heartbeat status (e.g., it is used in quite a few places in vim25/types/types.go). 

What about renaming it to "YellowHeartbeat"?

Reply:
Sure